app/javascript/mastodon/features/ui/components/navigation_panel.jsx

```diff
@@ -48,9 +45,9 @@ class NavigationPanel extends Component {
     return match || location.pathname.startsWith('/public');
   };
 
-  render () {
+  render() {
     const { intl } = this.props;
-    const { signedIn, disabledAccountId } = this.context.identity;
+    const { signedIn, disabledAccountId } = this.context;
```

Comment:
I wonder if we could keep `this.context.identity`, this is more explicit where those come from.
Ultimately we wont use class-based components so it does not matter, but it would make it clear that this is identity-related.

I dont think the React API handles this, but the context could be `identity: { … }`, and `useIdentity` could be `useContext(IdentityContext).identify;`

It makes the context shape more complex, but make consumer components more readable